CalculateCurrentIssues.cls

```diff
@@ -87,19 +87,19 @@ Issue__c TestNewIssue = new Issue__c
          pricebook__c            = 'CIO (Online)',
          deadline__c             = date.today().addDays(2)
          );
-         insert TestNewIssue;
+         insert testNewIssue;
 
 CalculateCurrentIssues.ClearAllIssueStatuses();
 
-list<issue__c> current_issues = [select id from issue__c where issue_status__c != ''];
-system.debug(current_issues.size());
-system.assertequals(0,current_issues.size());
+list<issue__c> currentIssues = [select id from issue__c where issue_status__c != ''];
+system.debug(currentIssues.size());
+system.assertequals(0,currentIssues.size());
 
 }
 
 public static testMethod void TestSetNewIssueStatuses() {
```

Comment:
* Test methods must use the `lowerCamelCase` naming convention. Please rename accordingly.
* I suggest to annotate test methods with `@IsTest` for better readability.